app/models/post.rb

```diff
@@ -1,9 +1,9 @@
 class Post < ActiveRecord::Base
 
-  has_many :received_requests, class_name: 'Request'
+  has_many :received_requests, class_name: 'Request', dependent: :destroy
   belongs_to :author,  class_name: 'User', foreign_key: :author_id
 
-end
-# test
+  validates :subject, :content, :author_id, presence: true
```

Comment:
validates :postal_horn: :post_office: :postal_code :city_sunrise: :city
so: to be clear: we should validate city and address presence is true, right?